graphistry/umap_utils.py

```diff
@@ -73,22 +74,22 @@ def is_legacy_cuml():
         return False
 
 
-UMAPEngineConcrete = Literal["cuml", "umap_learn"]
+UMAPEngineConcrete = Literal[f'{CUML}', f'{UMAP_LEARN}']
```

Comment:
Literal[..] worked when simple strings, not sure how to think why this is failing Liskov-SP?